docs/src/pages/customization/how-to-customize/how-to-customize.md

```diff
@@ -206,3 +206,22 @@ If you just want to add some global baseline styles for some of the HTML element
 If you are already using the [CssBaseline](/components/css-baseline/) component for setting baseline styles, you can also add these global styles as overrides for this component. Here is how you can achieve the same by using this approach.
 
 {{"demo": "pages/customization/how-to-customize/OverrideCssBaseline.js", "iframe": true, "height": 100}}
+
+> Note: It is a good practice to hoist the `<GlobalStyles />` to a static constant, to avoid rerendering. This will ensure that the `<style>` tag generated would not change on each render.
```

Comment:
`change` feels a bit mislead to me.

```suggestion
> Note: It is a good practice to hoist the `<GlobalStyles />` to a static constant, to avoid rerendering. This will ensure that the `<style>` tag generated would not recalculate on each render.
```